Tetris/src/tetris/GameRunner.java

```diff
@@ -30,20 +47,90 @@ static void startGame(){
 	 * @param user input
 	 */
 	static void inputListener(){
-		// move
+		// Listener: Press A
+			// Behavior: Rotate counterclockwise
+		// Listener: Press D
+			// Behavior: Rotate clockwise
 		
-		// rotate
+		// Listener: Press Left arrow
+			// Behavior: Shift Tetromino left 
 		
-		// drop
+		// Listener: Press Right arrow
+			// Behavior: Shift Tetromino right
 		
-		// pause
```

Comment:
Red is stuff I removed.  This is an older diff between a prior commit and the new code. If you look at the new code vs master you should see it properly